PICS/Resources/Localizable.xcstrings

```diff
@@ -465,6 +531,9 @@
           }
         }
       }
+    },
+    "This task has been completed!" : {
```

Comment:
Maybe changed this to a localized string so that it is easier to translate in German